src/components/header/header.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { StyledHeader } from "./header.styles";
+
+const Header = ({ text, back }) => {
```

Comment:
The code looks good! Add this header component to one of the main pages so we can see it show up when we run the app. Then we can style it and tweak it from there